syncd/NotificationQueue.h

```diff
@@ -0,0 +1,45 @@
+#pragma once
+
+#include "syncd.h"
```

Comment:
Can you remove this `include`?

Reply:
it's actually required since includes sai.h and defines _In_ macros used here

Reply:
i removed this include in https://github.com/Azure/sonic-sairedis/pull/541, introducing sal.h and adding it to swss-common with related PR

Reply:
If you need sai.h, just include the smaller one.

---
In reply to: [352917947](https://github.com/Azure/sonic-sairedis/pull/539#discussion_r352917947) [](ancestors = 352917947)

Reply:
done